tfexec/exit_errors.go

```diff
@@ -18,10 +18,18 @@ var (
 
 	usageRegexp = regexp.MustCompile(`Too many command line arguments|^Usage: .*Options:.*|Error: Invalid -\d+ option`)
 
-	noInitErrRegexp = regexp.MustCompile(`Error: Could not satisfy plugin requirements|` +
-		`Error: Could not load plugin|` + // v0.13
-		`Please run \"terraform init\"|` + // v1.1.0 early alpha versions (ref 89b05050)
-		`run:\s+terraform init`) // v1.1.0 (ref df578afd)
+	noInitErrRegexp = regexp.MustCompile(
+		// UNINITIALISED PROVIDERS/MODULES
+		`Error: Could not satisfy plugin requirements|` +
```

Comment:
gofmt and I are of differing opinions as to the best indentation for this section of code.